build.gradle

```diff
@@ -9,10 +9,6 @@ plugins {
 }
 
 repositories {
-    mavenLocal()
-    maven {
-        url = uri('https://repo.maven.apache.org/maven2')
```

Comment:
Removing this line breaks everything as this is the central maven repository from where everything is fetched. Test before sending PR. In this case, the test would be whether gradlew bootJar is passing successfully.